server/db/driver/pg/orders.go

```diff
@@ -0,0 +1,584 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package pg
+
+import (
+	"database/sql"
+	"encoding/hex"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/decred/dcrdex/server/account"
+	"github.com/decred/dcrdex/server/db"
+	"github.com/decred/dcrdex/server/db/driver/pg/internal"
+	"github.com/decred/dcrdex/server/market/types"
+	"github.com/decred/dcrdex/server/order"
+	"github.com/lib/pq"
+)
+
+type Error string
+
+func (err Error) Error() string { return string(err) }
+
+const (
+	ErrUnknownOrder = Error("unknown order")
+)
+
+// utxo implements order.UTXO
```

Comment:
`order.UTXO` is just

```go
type UTXO interface {
	TxHash() []byte
	Vout() uint32
}
```
since that's all it needs.

An integration test can check that `order.UTXO` fully overlaps with `asset.UTXO`.

Reply:
I could see naming `order.UTXO` to `order.Outpoint` instead just to differentiate too.

Reply:
Good suggestion. Done in https://github.com/decred/dcrdex/pull/47/commits/a410cffbb2dd1f272af17c9c07e30247001cc796